ajax/libs/react-datetime/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "react-datetime",
+  "version": "2.8.9",
+  "filename": "react-datetime.min.js",
+  "description": "A lightweight but complete datetime picker React.js component.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/YouCanBookMe/react-datetime.git"
+  },
+  "keywords": [
+    "react",
+    "react-component",
+    "datepicker",
+    "timepicker",
+    "datetimepicker",
+    "datetime"
+  ],
+  "author": "Javier Marquez",
+  "license": "MIT",
+  "npmName": "react-datetime",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    },
+    {
+      "basePath": "css",
```

Comment:
`css` is not suitable to be `basePath`. Please Use 
> "basePath": ""
"files": [
 "css/react-datetime*"
]
(Please ignore the indentation here)